cmd/fpga_plugin/dfl_test.go

```diff
@@ -215,7 +226,24 @@ func TestGetRegionTreeDFL(t *testing.T) {
 			Permissions:   "rw",
 		},
 	}
-	expected.AddDevice(regionMode+"-ce48969398f05f33946d560708be108a", "region2", dpapi.NewDeviceInfo(pluginapi.Healthy, nodes, nil, nil, nil, hooks))
+	cdiSpec = &cdispec.Spec{
+		Version: "0.5.0",
+		Kind:    dpapi.CDIVendor + "/fpga",
+		Devices: []cdispec.Device{
+			{
+				Name: "region2",
+				ContainerEdits: cdispec.ContainerEdits{
+					Hooks: []*cdispec.Hook{
+						{
+							HookName: HookName,
+							Path:     HookPath,
+						},
+					},
+				},
+			},
+		},
+	}
+	expected.AddDevice(regionMode+"-ce48969398f05f33946d560708be108a", "region2", dpapi.NewDeviceInfo(pluginapi.Healthy, nodes, nil, nil, nil, cdiSpec))
```

Comment:
Looks good to me now, thanks! You can convert this PR from `draft` to `ready for review` I guess.

Reply:
https://github.com/intel/intel-device-plugins-for-kubernetes/pull/1810 was the original one with the CDI restructuring. This PR is adding the GPU support on top of the #1810.